app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts

```diff
@@ -254,15 +254,58 @@ export default class RemoteBackend extends Backend {
 
   /** Invite a new user to the organization by email. */
   override async inviteUser(body: backend.InviteUserRequestBody): Promise<void> {
-    const path = remoteBackendPaths.INVITE_USER_PATH
```

Comment:
why remove `path`?

Reply:
AFAIR I inlined the varible

Reply:
yeah i mean that works, but separate `path` just makes it consistent with all the other endpoints in this file